.azure/pipelines/ci.yml

```diff
@@ -219,7 +220,6 @@ stages:
         -pack
         -noBuildNodeJS
         -noBuildJava
-        /bl:artifacts/log/build.win-arm.binlog
```

Comment:
@wtgodbe why doesn't this command get the `-nobl` addition? Same question for about 4 other steps in this file.

Reply:
Because it doesn't pass `-ci` - I did a global search for everywhere this file sets `-ci` and added `-nobl` in those locations

Reply:
Suggest that we should always pass `-ci` for consistency when building on the CI. I'll take that as part of the many changes I need to make in this PR.

Reply:
What does -ci also imply? Does it imply tests too?

Reply:
`-ci` changes the default `dotnet` install location and disabled node reuse. All good things.